sourcerer-esjc/src/main/java/org/elder/sourcerer/esjc/EventStoreEsjcEventRepository.java

```diff
@@ -475,10 +469,7 @@ private T normalizeEvent(final T rawEvent) {
         } catch (ExecutionException ex) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.elder.org/coding_rules#rule_key=squid%3AS1166)